index.html

```diff
@@ -0,0 +1,132 @@
+            malaria, disinfectants, and other applied science.
+          </li>
+          <li>
+            <span class="year">1942</span> - Receives a Ph.D. in Genetics and
+            Plant Pathology
+          </li>
+          <li>
+            <span class="year">1944</span> - Rejects a 100% salary increase from
+            Dupont, leaves behind his pregnant wife, and flies to Mexico to head
+            a new plant pathology program. Over the next 16 years, his team
+            breeds 6,000 different strains of disease resistent wheat -
+            including different varieties for each major climate on Earth.
+          </li>
+          <li>
+            <span class="year">1945</span> - Discovers a way to grown wheat
+            twice each season, doubling wheat yields
+          </li>
+          <li>
+            <span class="year">1953</span> - crosses a short, sturdy dwarf breed
+            of wheat with a high-yeidling American breed, creating a strain that
+            responds well to fertilizer. It goes on to provide 95% of Mexico's
+            wheat.
+          </li>
+          <li>
+            <span class="year">1962</span> - Visits Delhi and brings his
+            high-yielding strains of wheat to the Indian subcontinent in time to
+            help mitigate mass starvation due to a rapidly expanding population
+          </li>
+          <li>
+            <span class="year">1970</span> - receives the Nobel Peace Prize
+          </li>
+          <li>
+            <span class="year">1983</span> - helps seven African countries
+            dramatically increase their maize and sorghum yields
+          </li>
+          <li>
+            <span class="year">1984</span> - becomes a distinguished professor
+            at Texas A&M University
+          </li>
+          <li>
+            <span class="year">2005</span>- states "we will have to double the
+            world food supply by 2050." Argues that genetically modified crops
+            are the only way we can meet the demand, as we run out of arable
+            land. Says that GM crops are not inherently dangerous because "we've
+            been genetically modifying plants and animals for a long time. Long
+            before we called it science, people were selecting the best breeds."
+          </li>
+          <li><span class="year">2009</span>2009 - dies at the age of 95.</li>
+        </ul>
+      </div>
+      <div class="quote">
+        <p>
+          "Borlaug's life and achievement are testimony to the far-reaching
+          contribution that one man's towering intellect, persistence and
+          scientific vision can make to human peace and progress."
+        </p>
+        <span> -- Indian Prime Minister Manmohan Singh </span>
+      </div>
+      <div id="footer">
+        <p>
+          If you have time, you should read more about this incredible human
+          being on his <a href="#" target="_blank">Wikipedia entry.</a>
+        </p>
```

Comment:
Kindly put the right link for Wikipedia.

Reply:
@MoRaad97, Nice doing the review 👍, but on this comment, it will be helpful to mention why the changes you requested are necessary.